Formula/php.rb

```diff
@@ -0,0 +1,343 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net/"
+
+  url "https://php.net/get/php-7.2.0.tar.gz/from/this/mirror"
+  sha256 "801876abd52e0dc58a44701344252035fd50702d8f510cda7fdb317ab79897bc"
+
+  depends_on "argon2"
+  depends_on "aspell"
+  depends_on "curl" if MacOS.version < :lion
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "gmp"
+  depends_on "httpd"
+  depends_on "icu4c"
+  depends_on "imap-uw"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libpq"
+  depends_on "libsodium"
+  depends_on "libzip"
+  depends_on "net-snmp"
+  depends_on "openssl"
+  depends_on "pcre"
+  depends_on "unixodbc"
+  depends_on "webp"
+
+  needs :cxx11
+
+  def install
+    inreplace "configure",
+              "APACHE_THREADED_MPM=`$APXS_HTTPD -V | grep 'threaded:.*yes'`",
+              "APACHE_THREADED_MPM="
+
+    inreplace "sapi/apache2handler/sapi_apache2.c",
+              "You need to recompile PHP.",
+              "Homebrew PHP does not support a thread-safe php binary. "\
+              "To use the PHP apache sapi please change "\
+              "your httpd config to use the prefork MPM"
+
+    ENV.cxx11
+
+    # Fix missing header file during configure for libzip include
+    ENV.append_to_cflags "-I#{Formula["libzip"].opt_prefix}/lib/libzip/include"
+
+    config_path = etc/"php/#{php_version}"
+    ENV["lt_cv_path_SED"] = "sed"
+
+    args = %W[
+      --prefix=#{prefix}
+      --localstatedir=#{var}
+      --sysconfdir=#{config_path}
+      --with-config-file-path=#{config_path}
+      --with-config-file-scan-dir=#{config_path}/conf.d
+      --mandir=#{man}
```

Comment:
what happens without this?

Reply:
Without it, it defaults to the "PHP" layout and puts it in `prefix/php/man`. Perhaps we should consider using `--with-layout=GNU` instead and adjust to the different extension directories.